src/components/Calendar/Calendar.js

```diff
@@ -1,15 +1,14 @@
-import {React, useState} from 'react'
+import { React, useState } from 'react'
 import { Calendar, dateFnsLocalizer } from 'react-big-calendar'
-import 'react-big-calendar/lib/css/react-big-calendar.css'
-import './calendar.css'
-import { events ,resourceMap } from './data'
-import {customWeekViewEvent, customResourceViewEvent} from './CustomEvent'
+import '../../less/calendar.less'
```

Comment:
Changing the style import from 2 CSS files to 1 LESS file is the only actual change in this file; everything else is just automagic whitespace changes by the linter.